### PR TITLE
fix: Removing is_ticketing_enabled from event models

### PR DESCRIPTION
**Pull request by kushthedude**
Refers https://github.com/fossasia/open-event-frontend/issues/3664

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Don't drop the column please, just remove the field from schema and logic for it

**Comment by kushthedude** [comment]
@iamareebjamal Please have a look

**Comment by iamareebjamal** [comment]
Fix dredd tests

**Comment by kushthedude** [comment]
@iamareebjamal Why the `module` is getting treated as `None Type` in Dredd Tests ?

**Comment by iamareebjamal** [comment]
Because modules are not added in the hooks

File: app/api/events.py [hunk 1]
```diff
@@ -56,8 +56,8 @@ def validate_event(user, modules, data):
     if not user.can_create_event():
         raise ForbiddenException({'source': ''},
                                  "Please verify your Email")
-    elif data.get('is_ticketing_enabled', True) and not modules.ticket_include:
-            raise ForbiddenException({'source': '/data/attributes/is-ticketing-enabled'},
+    elif not modules.ticket_include:
+            raise ForbiddenException({'source': ''},
                                      "Ticketing is not enabled in the system")
```
Comment (mrsaicharan1) [inline app/api/events.py]: "This event doesn't have any tickets"

**Comment by kushthedude** [comment]
Ticketing is not enables. Its corret error message

**Comment by mrsaicharan1** [comment]
Okay.

**Comment by kushthedude** [comment]
@iamareebjamal Please review and merge

**Pull request merged**